src/index.js

```diff
@@ -76,7 +76,8 @@ class Backend {
         } catch (e) {
           return callback(`failed parsing ${url} to json`, false);
         }
-      });
+      })
+      .catch(() => callback(`failed loading ${url}`, false));
```

Comment:
This `catch` applies to the following scenarios:

1. The rejection of `fetch`.
2. A rejection or error in the first `then` call.
3. A rejection or error in the second `then` call.

However the callback message `failed loading for ${url}` applies to the first scenario.